pubspec.yaml

```diff
@@ -11,4 +11,11 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
-  pedantic: ^1.9.0
+  pedantic: ^1.8.0
```

Comment:
I had to lower pedantic's version in order for version solving to work with flutter_test.